### ISSUE TITLE
As a CIF maintainer, I want to have safeguards for file uploads, so that the app is not vulnerable to abuse

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-cif#1529]
#### Description:

We need basic safeguards so that the external upload mechanics can't be abused:
- Max attachment size
- file extension allow-listing (doc, docx, xls, xlsx, pdf, txt, csv, ... ?)
There needs to be a short discussion about what files the business area typically handles

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

?**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by Sepehr-Sobhani** [comment]
I think it would be nice if we can have a help text for users to know these safeguards before uploading any attachments.

**Comment by pbastia** [comment]
@Sepehr-Sobhani good point - the wording of that will need to be included in the UI Design of the upload feature @suhafa

**Comment by pbastia** [comment]
I know we've only implemented file upload for internal users, but we can't guarantee they won't use a compromised device. Implementing this would reduce our attack surface.
